registrations/permissions.py

```diff
@@ -76,6 +78,32 @@ def has_permission(self, request: Request, view: APIView) -> bool:
 
         return request.method in permissions.SAFE_METHODS
 
+
+class CanAccessSignup(SignUpBasePermission):
+    @staticmethod
+    def _has_object_update_permission(request: Request, obj: SignUp) -> bool:
+        if request.user.is_superuser or request.user.is_registration_admin_of(
+            obj.publisher
+        ):
+            return True
+
+        if obj.registration.registration_user_accesses.filter(
+            email=request.user.email
+        ).exists():
+            # Registration user can only change presence_status and nothing else,
+            # and they need to be strongly identified.
+            data_keys = set(request.data.keys())
+            return request.user.is_strongly_identificated and (
```

Comment:
`is_strongly_identificated` was renames as `is_strongly_identified` in https://github.com/City-of-Helsinki/linkedevents/pull/715